src/analysis/modifications/generic.jl

```diff
@@ -25,7 +25,7 @@ change_constraint(id::String, lb, ub) =
     end
 
 """
-    change_objective(new_objective::Union{String,Vector{String}}; weights=[], sense=MOI.MAX_SENSE)
+    change_objective(new_objective::Union{String,Vector{String}}; weights=[], sense=COBREXA.MOI.MAX_SENSE)
```

Comment:
we've got our aliases for these, `src/base/constants.jl` names them as MAX_SENSE and MIN_SENSE without any prefix business

Reply:
fixed

Reply:
without hte COBREXA prefix actually. We removed lots of that in the other PR :D